src/s3ql/backends/b2/b2_backend.py

```diff
@@ -83,10 +81,9 @@ def __init__(self, options):
         self.disable_versions = 'disable-versions' in self.options
         self.retry_on_cap_exceeded = 'retry-on-cap-exceeded' in self.options
 
-        # Test modes
-        self.test_mode_fail_some_uploads = 'test-mode-fail-some-uploads' in self.options
-        self.test_mode_expire_some_tokens = 'test-mode-expire-some-tokens' in self.options
-        self.test_mode_force_cap_exceeded = 'test-mode-force-cap-exceeded' in self.options
+        self._extra_headers = CaseInsensitiveDict()
+        if 'test-mode' in self.options:
+            self._test_headers['X-Bz-Test-Mode'] = self.options['test-mode']
```

Comment:
```python
            self._extra_headers['X-Bz-Test-Mode'] = self.options['test-mode']
```

Reply:
Thanks! Good catch. I really wish I could run tests for this backend.